pkg/inngest/batch.go

```diff
@@ -23,8 +23,8 @@ func NewEventBatchConfig(conf map[string]any) (*EventBatchConfig, error) {
 		return nil, fmt.Errorf("failed to decode batch config: %v", err)
 	}
 
-	if config.MaxSize > consts.MaxBatchSize {
-		config.MaxSize = consts.MaxBatchSize
+	if config.MaxSize <= 0 {
```

Comment:
Can we still add a max size?  We probably don't want to increase above 100 yet, and if we do it might be for new billing plans.

This specifiically let a customer cut costs by 100x (on inngest AND Vercel) lol

Reply:
I'm going to make changes on the cloud side to make it easy to override.
This part of the code is the early validation when a function registers, and if we override here, we won't know what the user intended vs what is the actual restriction, so this provides better visibility as well.

e.g.
Oh, I see you set 100000 in your function config, but your account only allows 100. that's why it's only 100.